### ISSUE TITLE
How can i open a .structures folder.

### ISSUE THREAD

**Issue by brondonychocmans** [ryandw11/CustomStructures#80]
https://github.com/ryandw11/CustomStructures/wiki/Structure-Formatting#replacement_blocks

(sorry for the broken english)

The link up bellow shows in a very simple way "how to open the demo.schem folder and configure it" But in the practice is almost impossible.

How can i learn step by step how to use this pluggin, im very confused tryng to get info but the main page skips steps and is very hard to follow.

i already created a structure ingame but no clue how to configure it, eaven the demo.shem is getting very hard to configure and of course, it dosent spawn naturaly... Im using NBTexplorer and Visual studio code, no one of them seems to be usefull...

is there any discord server where i can be assisted to open, configure and execute this pluggin?

**Comment by ryandw11** [comment]
The demo.schem isn't what you should be configuring. The `.schem` file stores a schematic. What you want to configure is a structure. A schematic is just simply a file contains all of your blocks. A structure is what stores the information of where the schematic should go. Structures are a yaml `.yml` file. You can find them in the structures folder. If you did not create a structure you must do that first by running the command listed [here](https://github.com/ryandw11/CustomStructures/wiki/Structure-Formatting#creating-a-new-structure) in game. 

You can use [this](https://discord.gg/Req5uE) discord server for further help if you need it. (under support go to #ryan-general)

**Comment by brondonychocmans** [comment]
the invitation expired! ;( can u send it again pls?

**Comment by ryandw11** [comment]
https://discord.gg/EPyj9Va

**Comment by ryandw11** [comment]
resolved.
